docs/core-concepts/validation-suites.mdx

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: 'Validation Suites'
+---
+
+# Validation Suites
+
+**Validation Suites** help ensure your game is functioning as intended by verifying that specific criteria are met
+over the course of a [Gameplay Session](gameplay-sessions/gameplay-sessions-getting-started).
+A Validation Suite is composed of a collection of [Scenarios](#validation-scenarios)
+and [Rules](#validation-rules). Validation Suites are managed through a dashboard in the
+[Regression Games web interface](https://play.regression.gg/validations).
+
+## Creating a Validation Suite
+
+Create a Validation Suite from the [Validation Suites dashboard](https://play.regression.gg/validations)
+by clicking the **Create Suite** button at the top of the page.
+Enter a name and optional description for your Suite, and choose a Base Gameplay Session.
+This Gameplay Session will be used to create your Scenarios,
+which can then be run against other Gameplay Sessions in the future for regression testing.
+
+![Create a Validation Suite](img/validation-suites/create-validation-suite.gif)
+
+## Validation Scenarios
+
+A **Validation Scenario** defines a functional test case by asserting that specific actions are taken during
+a Gameplay Session and that they produce expected changes in the game state.
+Steps can assert that a game scene is loaded, wait for mouse and keyboard inputs, check for the existence
+of specific game objects, and much more.
+
+Validation Scenarios execute in the same order as they appear in the Validation Suite dashboard, but order does not impact how Scenarios run.
```

Comment:
This is the first part that I felt confused. If scenarios execute in the order, then in my mind the order DOES impact how scenarios are run? 

Is it that the order doesn't impact whether the scenarios run? If this is not it, I don't think the text conveyed to me in how they don't impact one another. (or is it that because they run sequentially, they don't interfere with one another?)

Reply:
Oh... I see, this is about multiple Scenarios, not the steps in the scenarios. To clear this up consider adding a priming sentence that goes something like "You can have multiple scenarios in ..."

Reply:
Added that primer sentence and clarified that scenarios won't impact each other